yb-voyager/src/monitor/monitor_target_health.go

```diff
@@ -70,19 +68,17 @@ type MonitorTargetYBHealth struct {
 
 	//Display function for the printing any msg on the console
 	displayMsgFunc func(string)
-
-	tconf tgtdb.TargetConf
 }
 
-func NewMonitorTargetYBHealth(yb TargetDBForMonitorHealth, skipDiskUsageHealthChecks utils.BoolStr, skipReplicationChecks utils.BoolStr, skipNodeHealthChecks utils.BoolStr, displayMsgFunc func(msg string), ybClient YugabyteDBClient, tconf tgtdb.TargetConf) MonitorTargetYBHealth {
+func NewMonitorTargetYBHealth(yb TargetDBForMonitorHealth, skipDiskUsageHealthChecks utils.BoolStr, skipReplicationChecks utils.BoolStr, skipNodeHealthChecks utils.BoolStr, ybClient YugabyteDBClient, displayMsgFunc func(msg string)) MonitorTargetYBHealth {
```

Comment:
we should avoid using Boolstr outside the command layer (specifically when accepting inputs for a flag). For downstream code, plain booleans make more sense.